recipes/fastparquet/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: fastparquet
+  version: "0.0.1post2"
+
+source:
+  fn: fastparquet-0.0.1.post2.tar.gz
+  url: https://pypi.io/packages/source/f/fastparquet/fastparquet-0.0.1.post2.tar.gz
```

Comment:
Would recommend using jinja templates for versions to making updating more smooth. Please see the example recipe for details.